services/clusters-keeper/src/simcore_service_clusters_keeper/_meta.py

```diff
@@ -35,7 +36,11 @@ def get_summary() -> str:
 
 
 SUMMARY: Final[str] = get_summary()
-
+PACKAGE_DATA_FOLDER: Final[Path] = Path(
```

Comment:
Not going to adapt packages/service-library/src/servicelib/utils_meta.py and use it here??

Reply:
My usual, is to move things once I need them in at least 2 services...